ui/externs/ui.js

```diff
@@ -109,6 +111,14 @@ shaka.extern.UIVolumeBarColors;
  *   should be part of the UI.
  * @property {boolean} customContextMenu
  *   Whether or not a custom context menu replaces the default.
+ * @property {boolean} fastForwardOnTaps
```

Comment:
If you have implemented the ui.Element interface, why not configure them the way you do other elements, for example, in the `controlPanelElements` array?

Reply:
We would want to keep the fast-forward and rewind buttons linked though, wouldn't we? Technically, I think we could use a factory like how other elements are configured.

Also, I might be wrong but I do not think that will work since we need to add these elements to `controlsContainer_`, yet, the parent element for the factory is `controlsButtonPanel_`.